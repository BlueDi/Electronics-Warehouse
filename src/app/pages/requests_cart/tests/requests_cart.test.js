import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';
import RequestsCartList from '@pages/requests_cart';

describe('RequestsCartList', () => {
  it('should be defined', () => {
    expect(RequestsCartList).toBeDefined();
  });

  it('renders properly', () => {
    const component = renderer.create(
      <Router>
        <RequestsCartList />
      </Router>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
