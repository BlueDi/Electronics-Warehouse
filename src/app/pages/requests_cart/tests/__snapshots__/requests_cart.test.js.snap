// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RequestsCartList renders properly 1`] = `null`;
